ansible/roles/debops.docker/tasks/manage_components.yml

```diff
@@ -0,0 +1,148 @@
+---
+- name: Get full PKI paths
+  set_fact:
+    docker__register_ca_path: '{{ docker__pki_path + "/" + docker__pki_realm
+                                  + "/" + docker__pki_ca }}'
+    docker__register_crt_path: '{{ docker__pki_path + "/" + docker__pki_realm
+                                   + "/" + docker__pki_crt }}'
+    docker__register_key_path: '{{ docker__pki_path + "/" + docker__pki_realm
+                                   + "/" + docker__pki_key }}'
```

Comment:
These variables should be defined in the `defaults/main.yml` file instead, you won't need a separate task to set them up.

Reply:
Should I add new `docker__pki_*_path` variables or just replace the `docker__pki_*` ones?

Reply:
I went with just replacing the docker__pki_* variables, that's probably the cleanest solution.